### PR TITLE
Switch target form to have labels next to controls and a few small fixes

### PR DESCRIPTION
**Pull request by cquiroz**
This PR converts the target forms to have the labels next to the field

image

### CODE REVIEW

File: common/src/main/scala/explore/components/HelpIcon.scala [hunk 1]
```diff
@@ -17,31 +17,29 @@ import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.common._
 
-final case class HelpIcon(id: Help.Id) extends ReactProps[HelpIcon](HelpIcon.component)
+final case class HelpIcon(id: Help.Id)
```
Comment (cquiroz) [inline common/src/main/scala/explore/components/HelpIcon.scala:20]: @rpiaggio I converted this to a functional component for "fun", Does it seem ok?

**Review by toddburnside — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/HelpIcon.scala (hunk 1 above)
Comment (rpiaggio) [reply common/src/main/scala/explore/components/HelpIcon.scala:20]: I wonder what your definition of fun is.

It looks great. I'm planning to generalize the implicit conversion into `react-common` by having `ReactProps` accept a functional component as parameter, but until we have that we have to manually provide the conversion.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/HelpIcon.scala (hunk 1 above)
Comment (toddburnside) [reply common/src/main/scala/explore/components/HelpIcon.scala:20]: It's a `fun`ctional component.

**Pull request merged**